db/migrate/20210222100817_create_audits.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAudits < ActiveRecord::Migration[6.0]
+  def change
+    create_table :audits do |t|
+      t.references :planning_application, null: false, foreign_key: true
+      t.references :user
+      t.jsonb :activity_type, default: [], null: false
```

Comment:
Why is this a `jsonb` field? As we're using it as an enum, I think it should be an `int`?